.travis.yml

```diff
@@ -27,16 +27,18 @@ jobs:
           - $HOME/.cargo/bin
       before_cache:
         - rm -rf "$TRAVIS_HOME/.cargo/registry/src"
-      env: CARGO_INCREMENTAL=0
+      env:
+        - CARGO_INCREMENTAL=0
+        - RUSTFLAGS="-D warnings"
       before_install:
         - rustup component add clippy rustfmt
         - sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install awscli
         - $(aws ecr get-login --no-include-email --region $AWS_REGION)
       script:
-        - cargo build --all-targets --locked
+        - cargo build --workspace --all-targets --locked
         # Unit Tests and Linting
         - cargo test
-        - cargo clippy --all --all-targets -- -D warnings
+        - cargo clippy --workspace --all-targets -- -D warnings
```

Comment:
is the `-D warnings` still needed here now that they are set with `RUSTFLAGS`?

Reply:
I just double checked the clippy documentation https://github.com/rust-lang/rust-clippy#travis-ci :
> Note that adding -D warnings will cause your build to fail if any warnings are found in your code. That includes warnings found by rustc (e.g. dead_code, etc.).

That means we don't need the RUSTFLAG at all. I am going to remove it and move the clippy command above the build command because clippy is faster than build which catches errors earlier.